### PR TITLE
Feature/routing setup

### PR DESCRIPTION
**Pull request by paulstgermain**
This pull request adds basic routing configuration, as well as adds some stub components onto which we can build the front-end portion of the expected Labby 2.0 features by using `react-router-dom`. Accompanying this routing setup, I've included the appropriate links to each stub component for which a route was built by adding `react-router-dom`'s `Link`s to the ANT Design NavBar Ryan implemented earlier in the day.

This keeps with the standards set so far by Ryan and myself and leaves us off at a good place from which to build out these features and stub out the API connections to the `labs-be`.

### CODE REVIEW

**Comment by frankfusco** [comment]
[FE route configuration](https://trello.com/c/6ysvGt2d/12-fe-route-configuration)

File: src/App.tsx [hunk 1]
```diff
@@ -1,9 +1,38 @@
...
+
+//* * Component Imports **/
 import NavigationHeader from "./components/Navigation/Navigation";
+import Attendance from "./components/Attendance/Attendance";
+import TeamBuilder from "./components/TeamBuilder/TeamBuilder";
+import LearnerSuccess from "./components/LearnerSuccess/LearnerSuccess";
+import FourthComponent from "./components/FourthComponent/FourthComponent";
+import Home from "./components/Home/Home";
 
 const App = (): any => (
```
Comment (ryan-hamblin) [inline src/App.tsx:15]: Not gonna block the PR over this but as I've been studying up on this, these types of components that return child elements should probably be `React.ReactNode` type

File: src/App.tsx [hunk 2]
```diff
@@ -1,9 +1,38 @@
...
-  <NavigationHeader />
+  <>
+    <NavigationHeader />
+    <Switch>
+      <Route exact path="/">
+        <Home />
```
Comment (ryan-hamblin) [inline src/App.tsx:20]: Again not a blocking issue, but I'd love ensure that we're following best practices with the Route component. 

example: `<Route to='some-location' component={ComponentName}>` 

I'm not sure which is better or more commonly used so I'll find out and can refactor later if necessary.

**Review by ryan-hamblin — APPROVED** [review]
This is excellent for scaffolding and includes the proper placeholders that we need to move forward with our other tickets. I want to research more into the React Types that come with React so that we're not getting into any anti-patterns using `any`. 

I also want to research common practices for React Router... how're is the world doing things these days? Wrapping components w/ the `Route` component? or passing them in as a `component` prop?

Nothing to be overly concerned about however.

**Pull request merged**